src/stackwalk.c

```diff
@@ -919,7 +919,12 @@ static void jl_rec_backtrace(jl_task_t *t) JL_NOTSAFEPOINT
         memset(&c, 0, sizeof(c));
      #if defined(_OS_LINUX_) && defined(__GLIBC__)
         __jmp_buf *mctx = &t->ctx.ctx.uc_mcontext->__jmpbuf;
+      #if defined(_CPU_ARM_)
+        // https://github.com/JuliaLang/julia/issues/47345#issuecomment-1293706173
```

Comment:
```suggestion
```

Reply:
What source would you like me to quote for this change?  I'd like to have some kind of a justification.

Reply:
I think saying that "The libunwind-arm.h header defines its own copy of ucontext_t instead of using the platform definition directly."